libbpf-tools/trace_helpers.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __TRACE_HELPERS_H
+#define __TRACE_HELPERS_H
+
+struct ksym {
+	long addr;
+	char name[0];
```

Comment:
please no, it's cumbersome API and will prevent optimization I've been talking about. Keep it as `const char *`.

Reply:
Oh, sorry for the complexity, I'll drop RB tree and  implement my original version but with the API we discussed. Thank you.

Reply:
@anakryiko I've update PR with the original version with the API we discussed. PTAL. Thank you.